module/proto/gravity/v1/confirm.proto

```diff
@@ -1,66 +1,41 @@
 syntax = "proto3";
 package gravity.v1;
 
-option go_package = "github.com/cosmos/gravity-bridge/module/x/gravity/types";
+import "gogoproto/gogo.proto";
 
-// ConfirmType is the cosmos type of a confirmation of a given object from the eth signers
-enum ConfirmType {
-  CONFIRM_TYPE_UNSPECIFIED = 0;
-  CONFIRM_TYPE_VALSET      = 1;
-  CONFIRM_TYPE_BATCH       = 2;
-  CONFIRM_TYPE_LOGIC       = 3;
-}
+option go_package = "github.com/cosmos/gravity-bridge/module/x/gravity/types";
 
-// ConfirmLogicCall
-// When validators observe a MsgRequestBatch they form a batch by ordering
-// transactions currently in the txqueue in order of highest to lowest fee,
-// cutting off when the batch either reaches a hardcoded maximum size (to be
-// decided, probably around 100) or when transactions stop being profitable
-// (TODO determine this without nondeterminism) This message includes the batch
-// as well as an Ethereum signature over this batch by the validator
-// -------------
+// ConfirmLogicCall ...
 message ConfirmLogicCall {
-  string invalidation_id      = 1;
+  bytes  invalidation_id      = 1 [(gogoproto.casttype) = "github.com/tendermint/tendermint/libs/bytes.HexBytes"];
```

Comment:
why use hexbytes here?

Reply:
because there weren't any docs on why the invalidation id was a plain string so I switched it for consistency

Reply:
But why the cast type part? this doesnt need to be hexbytes, we use proto json encoding anyways

Reply:
because it casts directly to hex bytes and it's easier to work with them. that's the only reason tbh

Reply:
I think this hurts compatibility with other languages so lets try to use the standard types?